src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -166,5 +166,12 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </remarks>
         /// </summary>
         public bool UseDefaultJsonSerialization { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Gets or sets the DateTime from which feed should start from. 
+        /// This is useful when you need to skip past and start from a checkpoint. It is best to use UTC date / time format to avoid any discrepencies.
+        /// </summary>
+        public string StartFrom { get; set; }
```

Comment:
We already have a StartFromBeginning, should this be StartFromTime ?

Reply:
StartFromTime is a fair option. I didn't want to restrict it to just time as its really a datetime :) . Let me know if you strongly feel the other way and I can make the change.

Reply:
We could use StartFromDateTime too, anything that is more precise than StartFrom

Reply:
updated